src/main/RPBuilder.ts

```diff
@@ -137,13 +156,14 @@ export default class RPBuilder {
     return this;
   }
 
-  addPresentationDefinitionClaim(definitionOpt: PresentationDefinitionWithLocation): RPBuilder {
-    if (!this.claims || !this.claims.presentationDefinitions) {
+  addClaims(definitionOpt: IPresentationDefinition, idToken?: IdTokenPayload): RPBuilder {
```

Comment:
Original method name was better, since JWTs are all about claims to begin with

Reply:
Does it make sense to keep the token_id in the claims? It's in the examples provided, however I cannot find anything in the specs to support sending it in the request as a claim.

Reply:
The naming issue is fixed